lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/controller/GraphController.java

```diff
@@ -291,12 +316,12 @@ private void repaint() {
             }
             view = new TileView(this);
 
-            wrapper.hvalueProperty().addListener(
+            scrollPane.hvalueProperty().addListener(
```

Comment:
tried to remove then, but the formatter aligns the code in a very strange way without the brackets.